### PR TITLE
add cisco secure endpoint analytic content

### PR DESCRIPTION
**Pull request by spsocprime**
Fixes #
Added analytic contetnt for Cisco Secure Endpoint Data Connector

### CODE REVIEW

**Review by aprakash13 — CHANGES_REQUESTED** [review]
The queries generally look good. However, for the  analytical rules  we need to add version number and kind in the end after entity mapping. Something like below.

version: 1.0.0
kind: Scheduled

See a detection rule as an example:  https://github.com/Azure/Azure-Sentinel/blob/master/Detections/ASimFileEvent/imFileESolarWindsSunburstSupernova.yaml

**Review by NikTripathi — CHANGES_REQUESTED** [review]
Workbook queries are failing. Please look into it. Thanks
image

**Comment by spsocprime** [comment]
> Workbook queries are failing. Please look into it. Thanks image

We've made PR for updating parser/function for this data connector(https://github.com/Azure/Azure-Sentinel/pull/3431). So I guess your functions does not have this fields which were added in the PR.
If you merge abovementioned PR it will fix the issue

**Review by NikTripathi — APPROVED** [review]
Workbook looks good. Can merge once validations passes.

File: Solutions/CiscoSecureEndpoint/Analytic Rules/CiscoSEWebshell.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+id: b6df3e11-de70-4779-ac9a-276c454a9025
```
Comment (NikTripathi) [inline Solutions/CiscoSecureEndpoint/Analytic Rules/CiscoSEWebshell.yaml:1]: There should not be 2 templates with the same ID, but the id b6df3e11-de70-4779-ac9a-276c454a9025 is duplicated.

**Review by NikTripathi — CHANGES_REQUESTED** [review]

**Comment by NikTripathi** [comment]
Since you are using the parser in queries, you will have to add its schema in json format under [CustomTable folder](https://github.com/Azure/Azure-Sentinel/tree/master/.script/tests/KqlvalidationsTests/CustomTables). Only than the validations would pass.

**Comment by vmanojreddy** [comment]
@spsocprime  checks failing due to CiscoSecureEndpoint table could not found in CustomTables folder.
Path refrence https://github.com/Azure/Azure-Sentinel/tree/master/.script/tests/KqlvalidationsTests/CustomTables

**Comment by NikTripathi** [comment]
@spsocprime Please update.

**Comment by NikTripathi** [comment]
@spsocprime please address my comments. Thanks

**Comment by NikTripathi** [comment]
@spsocprime Please resolve conflicts.

**Comment by spsocprime** [comment]
fixed

**Pull request merged**